components/inspectit-ocelot-configurationserver-ui/src/components/views/configuration/documentation/BaseDocsElement.js

```diff
@@ -25,6 +25,7 @@ const BaseDocsElement = ({ data, type, registerRef, scrollTo }) => {
           padding: 0.5rem;
           transition: background-color 0.2s;
           border-bottom: solid 1px #eeeeee;
+          white-space: pre-wrap;
```

Comment:
Depending on if this should only be possible for the main 'description' or for example also for descriptions of inputs in actions, the CSS statement could also be moved to .description instead of the whole .element